### DISCUSSION TITLE
Possibility of new projects ?

### DISCUSSION THREAD

**Discussion by aryanbhosale** [openclimatefix/community#7] [Ideas]
Hi community!

I don't know if this is the right category but I picked the one that matched the closest to it.

I just read up about the world's largest solar park in khavda by adani green.

Are there any projects OCF plans to do to forecast the data we can gather from there?

I think that something exciting is happening since I recently saw OCFs post on LinkedIn where Dan and Peter had travelled to New Delhi to discuss some exciting projects 👀

I would love to know more and get involved!

**Comment by zakwatts** [comment]
Hi @aryanbhosale! Large solar farms are definitely somewhere we would love to see our forecasts being used. And yes forecasting in India is very much on our radar at the moment. I'll try to give updates on this thread when we are able to.

**Reply by aryanbhosale** [reply]
That's really great @zakwatts ! I'd love to hear more about this whenever possible
